src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsInterstitialObject.java

```diff
@@ -16,6 +16,21 @@
  */
 public class AdsInterstitialObject extends AdsBaseObject {
 
+  /**
+   * Checks if the actual size (width or height) is within the "range of correctness" ;)
+   *
+   * We don't want to be pixel perfect in terms of checking the size of element because
+   * it depends on many things such as in example components of a browser's window.
+   *
+   * We also don't want to just check if the content of scalable interstitial is bigger
+   * than the original image. So, we came up with a reasonable size difference margin.
+   *
+   * The size defined here is doubled at the end.
+   *
+   * @see AdsInterstitialObject::isSizeCorrect()
+   */
+  private final static int SIZE_DIFFERENCE_TOLERANCE = 32;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)